tensorflow/python/eager/benchmarks_test.py

```diff
@@ -1620,5 +1627,42 @@ def benchmark_tf_range_return_int32_GPU(self):
   def benchmark_tf_range_return_int64_GPU(self):
     self._benchmark_tf_range_return(dtype=dtypes.int64, device=GPU)
 
+  def _benchmark_embedding_lookup_sparse_sparse_input(self, batch_size=32000, device=GPU):
+    def func(sp_ids):
+      return embedding_ops.embedding_lookup_sparse(self._m_10000_by_16, sp_ids, None)
+
+    with context.device(device):
+      values = random_ops.random_uniform(shape=(batch_size,),
+                        minval=1,
+                        maxval=10000,
+                        dtype=dtypes.int64)
+      value_rowids = ops.EagerTensor(np.arange(batch_size), device=device)
+
+      ragged_input = ragged_tensor.RaggedTensor.from_value_rowids(values, value_rowids)
+      sparse_input = sparse_ops.from_dense(ragged_input.to_tensor())
+      func(sparse_input)
+      self._run(lambda: func(sparse_input), num_iters=2000)
+
+  def benchmark_tf_embedding_lookup_sparse_sparse_input(self):
+    self._benchmark_embedding_lookup_sparse_sparse_input()
```

Comment:
I think we don't necessarily need this one line function if we only use the default parameters.